patches/minecraft/net/minecraft/item/ItemArmor.java.patch

```diff
@@ -20,17 +40,22 @@
      public static enum ArmorMaterial
      {
          LEATHER("leather", 5, new int[]{1, 2, 3, 1}, 15, SoundEvents.field_187728_s, 0.0F),
-@@ -235,6 +248,8 @@
-         private final int field_78055_h;
+@@ -236,6 +253,10 @@
          private final SoundEvent field_185020_j;
          private final float field_189417_k;
-+        //Added by forge for custom Armor materials.
-+        public Item customCraftingMaterial = null;
  
++        //Added by forge for custom Armor materials.
++        @Nullable
++        public ItemStack repairMaterial = null;
```

Comment:
ItemStack in 1.11 must never be null or nullable. 
Use `ItemStack.field_190927_a` as a value here and remove the nullable annotation.
I made the same mistake in `Item.ToolMateral` and it needs to be fixed there too.